src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/ProjectsController.kt

```diff
@@ -86,6 +89,27 @@ class ProjectsController(private val session: Session,
         return SuccessResponse(project).asResponseEntity()
     }
 
+    @PostMapping("/project/{projectId}/metadata")
```

Comment:
Since it's called 'note' everywhere else, I'd probably be inclined to just call this endpoint 'note' too until we decide we need to add some other metadata to it too.